src/pywifes/wifes_wsol.py

```diff
@@ -299,43 +309,22 @@ def weighted_loggauss_arc_fit(subbed_arc_data,
             # MJI: with the following test, the code ran faster, but even just the Pool(cpu)
             # command slowed things down. 
             #jobs2 = [jobs[:len(jobs)//4], jobs[len(jobs)//4:2*len(jobs)//4],jobs[2*len(jobs)//4:3*len(jobs)//4],jobs[3*len(jobs)//4:]]

```

Comment:
Maybe we should remove this cemented-out code as `jobs` is not a given argument in `weighted_loggauss_arc_fit`.